releasey/README.md

```diff
@@ -0,0 +1,131 @@
+- GPG setup and key configuration
+- Maven/Gradle credentials
+- Git remote setup for Apache repository
+- GitHub token for test status verification
+
+### 2. Create Release Branch
+```bash
+./02-create-release-branch.sh --version 1.0.0-incubating [--commit HEAD] [--recreate]
+```
+Creates a release branch and sets the target release version:
+- Creates `release/x.y.z` branch from specified commit
+- Updates `version.txt` with the release version
+- Updates `CHANGELOG.md` using Gradle task
+- Pushes changes to Apache remote
+
+### 3. Create Release Candidate Tag
+```bash
+./03-create-release-candidate-tag.sh --version 1.0.0-incubating-rc1
+```
+Creates a release candidate tag for a release candidate:
+- Creates and pushes `apache-polaris-x.y.z-incubating-rcN` tag
+- Validates RC sequence (RC2+ requires previous RC to exist)
+- Checks out the created tag
+
+### 4. Build and Verify Tests
+```bash
+./04-build-and-test.sh
+```
+Builds Polaris and ensures all integration and regression tests have been run:
+- Performs clean build with Gradle
+- Verifies that all GitHub checks have passed for current commit
+- Ensures all CI workflows are successful before release
+
+### 5. Build and Stage Distributions
+```bash
+./05-build-and-stage-distributions.sh
+```
+Builds and stages release artifacts:
+- Must be run from a release candidate tag
+- Builds source and binary distributions
+- Stages artifacts to Apache dist dev repository
+- Publishes Maven artifacts to Apache staging repository
+
+### 6. Build and Stage Docker Images
+```bash
+./06-build-and-stage-docker-images.sh
+```
+Builds and publishes multi-platform Docker images:
+- Must be run from a release candidate tag
+- Builds polaris-server and polaris-admin Docker images
+- Publishes images to DockerHub with RC tag
+- Supports multi-platform builds (linux/amd64, linux/arm64)
+
+## Test Scripts
+
+These scripts are used to test the release automation without making actual changes:
+
+### Running Tests
+```bash
+# Test individual scripts
+./test/test-02-create-release-branch.sh
+./test/test-03-create-release-candidate-tag.sh
+./test/test-04-build-and-test.sh
+./test/test-05-build-and-stage-distributions.sh
+./test/test-06-build-and-stage-docker-images.sh
+```
+
+All test scripts run in dry-run mode and verify the exact commands that would be executed.
+
+## Environment Variables
+The releases script initialize environment variables if not already set.
+This means that it is possible to override the default values by setting the environment variables before running the scripts.
+Example:
+
+- `DRY_RUN=0` - Disable dry-run mode (default: enabled)
+- `APACHE_REMOTE_NAME=my-apache-remote` - Name of git remote where release branches and tags are pushed (default: "apache")
+- `KEYSERVER` - GPG keyserver URL (default: "hkps://keyserver.ubuntu.com")
+- ...
+
+See the content of `libs/_constants.sh` for the list of all environment variables and their default values.
+
+## Prerequisites
```

Comment:
nit: can we move this section to the top, so that users can set them before running any script?

Reply:
Fixing...